resources/assets/js/snipeit.js

```diff
@@ -464,3 +464,28 @@ $(document).ready(function () {
     });
 
 });
+
+
+/**
+ * Toggle disabled
+ */
+(function($){
+		
+    $.fn.toggleDisabled = function(callback){
```

Comment:
I feel like there should already be a jquery thing you can do to directly twiddle the disabled attribute. No? Still need this? Feels like we shouldn't need that, but I could be just misreading it.

Reply:
I couldn't find anything straight jQuery that would do it since i'm toggling two different fields. If found this function which works nicely.